packages/sdk-react/package.json

```diff
@@ -13,18 +13,18 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/FusionAuth/fusionauth-react-sdk.git"
+    "url": "https://github.com/FusionAuth/fusionauth-javascript-sdk"
   },
   "author": "FusionAuth",
   "license": "Apache",
   "bugs": {
-    "url": "https://github.com/FusionAuth/fusionauth-react-sdk/issues"
+    "url": "https://github.com/FusionAuth/fusionauth-javascript-sdk.git/issues"
```

Comment:
@JakeLo123 small change on the path here, remove `.git` so the url is `https://github.com/FusionAuth/fusionauth-javascript-sdk/issues`

Reply:
good catch, thank you